tests/unit/CalendarPickerTest.js

```diff
@@ -7,6 +7,7 @@ import DateUtils from '../../src/libs/DateUtils';
 const monthNames = DateUtils.getMonthNames(CONST.LOCALES.EN);
 
 jest.mock('@react-navigation/native', () => ({
+    ...jest.requireActual('@react-navigation/native'),
```

Comment:
How is this related to this PR?

Reply:
This fixes a typescript issue caused by creating a custom overlay https://github.com/Expensify/App/issues/33389#issuecomment-1877992036